src_c/display.c

```diff
@@ -2218,6 +2307,25 @@ pg_get_desktop_screen_sizes(PyObject *self, PyObject *args)
     return result;
 }
 
+static PyObject *
+pg_is_fullscreen(PyObject *self, PyObject *args)
```

Comment:
Looks great. :)

Should we extract this into a separate commit/branch? (It looks self-contained and a no-brainer to merge)

Reply:
I'd really rather not. I would like to have this (and also #1705) in the same release with the warnings. I don't want people to depend on undocumented behaviour and then file bugs about it.

Reply:
Ok, fine with me. :)